api/src/main/java/org/openmrs/module/fhir/api/strategies/observation/ObservationStrategy.java

```diff
@@ -228,6 +227,11 @@ public Observation updateFHITObservation(Observation observation, String uuid) {
 		List<Observation> obsList = new ArrayList<>();
 		if (codings.isEmpty()) {
 			return obsList;
+		} else if (codings.size() == 1 && codings.get(0).getModifier() == null && codings.get(0).getSystem() == null) {
```

Comment:
This strikes me as a bit hackish. What's the justification for separating out the case where the codings list consists of a single object versus something that works with 1+ objects?

Reply:
TokenParam - 5089|PIH 
TokenOrListParam - 5089|PIH,5084|CIEL,4583-5|LOINC
TokenOrListParam - 5089AAAAAAAAAAAAAAAAAAAAA

@ibacher Yes, I agree hackish. I feel it was necessary. I am trying to reuse code [here](https://github.com/openmrs/openmrs-module-fhir/pull/218/commits/98de31a08e16541a063f56a21d99d92552b17e30) I implemented before. How would you know that the user has pass the concept uuid not token with the system?
Unless maybe you create another endpoint instead of TokenOrListParam take string as a param...